Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -615,3 +611,15 @@ void Renderer::Swap(u32 xfbAddr, u32 fbWidth, u32 fbStride, u32 fbHeight, const
 	XFBWrited = false;
 }
 
+void Renderer::FlipImageData(u8* data, int w, int h, int pixel_width)
```

Comment:
Why don't we use swscale to do that? I'm pretty sure it can (I've used it for that in the past). It will be faster and more transparent in terms of API.